vstools/functions/funcs.py

```diff
@@ -183,8 +209,12 @@ def without_planes(self, planes: PlanesT) -> list[int]:
 
     def return_clip(self, processed: vs.VideoNode) -> vs.VideoNode:
         """
-        Function used at the end of the function, to convert back to original format and
-        optionally bitdepth, merge back chroma.
+        Convert the processed clip back to the original clip's format and merge back chroma if necessary.
+        If `bitdepth != None`, it will also dither back to the original bitdepth.
```

Comment:
dithering doesn't always happen, just say converted to depth like in the \_\_init\_\_